src/varianttur/index.tsx

```diff
@@ -0,0 +1,257 @@
+import React, { useState } from "react";
+import style from "./varianttur.module.css";
+import Head from "next/head";
+
+const and = (...classes: string[]) => classes.join(" ");
```

Comment:
I can see this useful as a utils function. Where should utility functions live, `src/utils`?

Reply:
Sure. Could move it as an util already. Usually, I move them in when they are needed (or even sometimes just duplicate, as it allows for easier deletion/cleanup and at low cost for such small helper functions). I think if we want them as utils we can do `src/utils/somenamehere.ts`. Don't know the name, though. But `react` maybe? At some point we could even collect `@variant/helpers`.